build.sbt

```diff
@@ -413,14 +413,28 @@ lazy val ratSettings = Seq(
 )
 
 lazy val unidocSettings = Seq(
-  ScalaUnidoc / unidoc / unidocProjectFilter := inProjects(sapi, udf),
+  ScalaUnidoc / unidoc / unidocProjectFilter := inProjects(sapi, udf, runtime1),
   ScalaUnidoc / unidoc / scalacOptions := Seq(
     "-doc-title",
     "Apache Daffodil " + version.value + " Scala API",
     "-doc-root-content",
     (sapi / baseDirectory).value + "/root-doc.txt"
   ),
-  JavaUnidoc / unidoc / unidocProjectFilter := inProjects(japi, udf),
+  ScalaUnidoc / unidoc / unidocAllSources :=
+    (ScalaUnidoc / unidoc / unidocAllSources).value.map { sources =>
+      sources.filter { source =>
+        //
+        // This filter limits the scaladoc to only files which have
+        // api, sapi, or japi as directory names in their paths.
+        //
+        val str = source.toString
+        str.contains("/udf/") ||
+        str.contains("/api/") ||
+        str.contains("/sapi/") ||
+        str.contains("/japi/")
```

Comment:
/japi/ doesn't exist in scaladoc. It doesn't hurt to have it I guess, and it makes the logic the same, so maybe this is fine?